src/dotnet/commands/dotnet-build/BuildCommandApp.cs

```diff
@@ -46,6 +48,8 @@ public class BuildCommandApp
         public bool ShouldNotUseIncrementality { get; set; }
         public bool ShouldSkipDependencies { get; set; }
 
+        public bool IsServiceable { get; set; }
```

Comment:
Is someone actually consuming this property and passing the value into `dotnet compile`?  I don't see that happening anywhere.